java-api/src/main/java/com/sap/cloud/security/token/Token.java

```diff
@@ -45,6 +44,15 @@ static Token create(String jwt) {
 		if (services.isEmpty()) {
 			throw new ProviderNotFoundException("No TokenFactory implementation found in the classpath");
 		}
+		if (services.size() > 2) {
+			throw new ProviderException("More than 2 TokenFactory service providers found. There should be only one");
+		}
+		if (services.size() == 2) {
+			return services.stream()
+					.filter(tokenFactory -> !tokenFactory.getClass().getName()
+							.equals("com.sap.cloud.security.servlet.HybridTokenFactory"))
+					.findFirst().get().create(jwt);
```

Comment:
would be cool, if we have a constant for that class name as part of the HybridTokenFactory class. Better for refactoring.

Reply:
that wouldn't be accessible in Token class thou

Reply:
ok